descriptor.md

```diff
@@ -19,7 +19,8 @@ The following fields contain the primary properties that constitute a Descriptor
 
 - **`mediaType`** *string*
 
-  This REQUIRED property contains the MIME type of the referenced content.
+  This REQUIRED property contains the media type of the referenced content.
+  Values MUST comply with [RFC 6838][rfc6838].
```

Comment:
Was there a particular normative portion of RFC 6838 that you feel does not apply to `mediaType` values?  For example, the [vendor tree][1] semantics are outside of the [naming requirements section][2], and I think both of those apply to `mediaType` values.

[1]: https://tools.ietf.org/html/rfc6838#section-3.2
[2]: https://tools.ietf.org/html/rfc6838#section-4.2